### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rybon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Tuil is a template that helps to get you started with React Native using TypeScr
 **Note this template supports react-native ^0.60.0 and react-native-cli ^2.0.0, because of breaking changes in the custom template mechanism.**
 
 ```bash
-npx react-native init MyApp --template=react-native-template-tuil
+npx react-native init MyApp --template=tuil
```
Comment (patrickkempff) [inline README.md]: This no longer works with the latest version of the react-native cli if i am correct?

Comment (rybon) [reply README.md]: Not sure. It couldn't resolve `--template=react-native-template-tuil`, but could resolve `--template=tuil`.

Comment (patrickkempff) [reply README.md]: Are you using the old CLI? It was an issue with the old CLI. Since the new CLI it is not possible anymore to pass a shorthand.

Comment (rybon) [reply README.md]: Ah ok, I'm using:

```
$ react-native -v
react-native-cli: 2.0.1
```

**Comment by patrickkempff** [comment]
Not needed anymore

**Pull request closed without merging**